contracts/test/helpers/CompoundGovernorTest.sol

```diff
@@ -17,32 +18,51 @@ contract CompoundGovernorTest is Test, CompoundGovernorConstants {
     CompoundGovernor.ProposalGuardian proposalGuardian;
     uint96 constant PROPOSAL_GUARDIAN_EXPIRY = 1_739_768_400;
 
+    // GovernorBravo to receive upgrade proposal
+    address constant GOVERNOR_BRAVO_DELEGATE_ADDRESS = 0xc0Da02939E1441F497fd74F78cE7Decb17B66529;
+    GovernorBravoDelegate public constant GOVERNOR_BRAVO = GovernorBravoDelegate(GOVERNOR_BRAVO_DELEGATE_ADDRESS);
+
+    // The deployed CompooundGovernor address for testing upgradability after deployment
```

Comment:
```suggestion
    // The deployed CompoundGovernor address for testing upgradability after deployment
```

Reply:
can you move this to our constants file? can we call it DEPLOYED_UPGRADE_CANDIDATE?